spec/configuration_spec.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'Flatirons::Saas.configure'  do
+  it 'should set the stripe api key' do
+    stripe_api_key = 'test'
+    Flatirons::Saas.configure do |config|
+      config.stripe_api_key = stripe_api_key
+    end
+
+    expect(Flatirons::Saas.stripe_api_key).to eq(stripe_api_key)
```

Comment:
does this work if I set stripe api key globally in a .env file?

Reply:
This strpe_api_key will be set by our projects in our initializer flatirons_saas.rb

This will be configured on any project:
```ruby
# frozen_string_literal: true

Flatirons::Saas.configure do |config|
  # Set stripe api key
  #
  # Example:
  config.stripe_api_key = ENV['STRIPE_API_KEY']
end
``
`